test/resources/v5/order-recipients.test.js

```diff
@@ -157,6 +158,15 @@ describe('v5 order-recipients', () => {
       });
 
       it('calls API server URL and returns the response to the callback if the create was successful', async () => {
+        // We want to simulate custom headers being injected into the request
+        jest.spyOn(ordersClient, 'post').mockImplementation((...params) => {
+          const instance = axios.create({
+            headers: { 'X-Custom-Header': 'foobar' },
+          });
+
+          return instance.post(...params);
+        });
+
```

Comment:
Can we leave the test as it already was and instead create a new one? I see this as 2 different scenarios. We should test for the case where the custom header is present, and for when it's not, I think